docs/components/sensor/_index.md

```diff
@@ -73,9 +73,13 @@ The [sensor API](/appendix/apis/components/sensor/) supports the following metho
 
 ## Troubleshooting
 
-You can find additional assistance in the [Troubleshooting section](/appendix/troubleshooting/).
+If your sensor is not working as expected, follow these steps:
 
-{{< snippet "social.md" >}}
+1. Check your machine logs on the **LOGS** tab to check for errors.
+2. Review your sensor model's documentation to ensure you have configured all required attributes.
+3. Click on the **TEST** panel on the **CONFIGURE** or **CONTROL** tab and test if you can use the sensor there.
```

Comment:
[opt] Not all sensors are actual hardware but hopefully people using sensor modules for software "sensors" will just skip that step
```suggestion
1. Check your machine logs on the **LOGS** tab to check for errors.
1. Review your sensor model's documentation to ensure you have configured all required attributes.
1. Check that any wires are securely attached to the correct pins.
1. Click on the **TEST** panel on the **CONFIGURE** or **CONTROL** tab and test if you can use the sensor there.
```